packages/client/hmi-client/src/components/responsive-matrix/matrix-grid.fs.glsl.ts

```diff
@@ -27,25 +30,27 @@ uniform float uGridRowDisplayTransition;
 
 out vec4 fragColor;
 
+${getMicroEl}
```

Comment:
Just a point for discussion: not a huge fan of storing shader files as a string ...it makes the code a bit hard to read. Are there better ways? I know you can probably use the `loader` and treat them as assets....albeit it is still a bit awkward.